### PR TITLE
2774 jen1

### PR DESCRIPTION
**Pull request by jenlampton**
PR for https://github.com/backdrop/backdrop-issues/issues/2774

### CODE REVIEW

File: core/modules/update/tests/themes/update_test_admintheme/update_test_admintheme.info [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = Update test admin theme
+description = Test theme which is used as admin theme.
+core = 7.x
```
Comment (quicksketch) [inline core/modules/update/tests/themes/update_test_admintheme/update_test_admintheme.info]: This should be `backdrop = 1.x` I believe.

File: core/modules/update/tests/update.test [hunk 2]
```diff
@@ -470,6 +469,55 @@ class UpdateTestContribCase extends UpdateTestHelper {
...
+        'hidden' => FALSE,
+      ),
+      'update_test_basetheme' => array(
+        'project' => 'update_test_basetheme',
+        'version' => '7.x-1.1',
+        'hidden' => FALSE,
+      ),
+      'update_test_subtheme' => array(
+        'project' => 'update_test_subtheme',
+        'version' => '7.x-1.0',
```
Comment (quicksketch) [inline core/modules/update/tests/update.test]: We should update all these `version` strings to be `1.x-` instead of `7.x-`.

**Review by quicksketch — CHANGES_REQUESTED** [review]
Needs a few very minor updates. Overall looks great!

**Comment by jenlampton** [comment]
Fixed, and rebased.

**Review by quicksketch — APPROVED** [review]

**Comment by quicksketch** [comment]
Thanks @jenlampton! Merged into 1.x and 1.8.x.

**Pull request merged**